src/App.jsx

```diff
@@ -1,44 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
```

Comment:
Great work, but here you also can use `import { Tabs } from './components/Tabs`;
and if  use this, you don`t need to change export in 'index.js':)